lib/beeswaxapi/request.rb

```diff
@@ -78,16 +78,21 @@ def request_for(**opts)
         end
 
         if response.success?
-          return success_response_handler(response)
+          success_response_handler(response)
         elsif response.timed_out?
-          return timed_out_response_handler(response)
+          timed_out_response_handler(response)
         elsif response.code >= 400 && response.code < 500
-          return failure_response_handler(response)
+          failure_response_handler(response)
         elsif response.code >= 500 && response.code
```

Comment:
why we have `response.code >= 500 && response.code`?

Reply:
I have no idea :)